app/package.json

```
@@ -3,22 +3,34 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
     "build": "next build",
+    "dev": "next dev",
+    "lint": "next lint",
     "start": "next start",
-    "lint": "next lint"
+    "test": "jest"
```

Let's pull in the [`--ci`](https://jestjs.io/docs/cli#--ci) and [`--coverage`](https://jestjs.io/docs/cli#--coverageboolean) flags from [ca ui](https://github.com/cagov/ui-claim-tracker/blob/600d52f82607fce202349bad8c9ccf69184e6d10/package.json#L10):
```suggestion
    "test": "jest --ci --coverage"
```

---

The reasoning for this change is:
- for `--ci`: It allows jest to be more easily run in a CI environment
- for `--coverage`: It provides code coverage statistics as part of running the tests